Source/WebCore/testing/Internals.cpp

```diff
@@ -1403,7 +1404,7 @@ Vector<Internals::AcceleratedAnimation> Internals::acceleratedAnimationsForEleme
 
 unsigned Internals::numberOfAnimationTimelineInvalidations() const
 {
-    return frame()->document()->timeline().numberOfAnimationTimelineInvalidationsForTesting();
+    return frame() ? frame()->document()->timeline().numberOfAnimationTimelineInvalidationsForTesting() : 0;
```

Comment:
I’d think this would need to be in a `RefPtr` or call `protectedFrame()` in our new safer C++ model.

Reply:
Done.